app/src/main/java/org/mozilla/fenix/composables/SelectableChip.kt

```diff
@@ -0,0 +1,83 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.composables
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.selection.selectable
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.capitalize
+import androidx.compose.ui.text.intl.Locale
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import mozilla.components.ui.colors.PhotonColors
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Default layout of a selectable chip.
+ *
+ * @param text [String] displayed in this chip. Ideally should only be one word.
+ * @param isSelected whether this should be shown as selected.
+ * @param onClick callback for when the user taps this.
+ */
+@Composable
+fun SelectableChip(
+    text: String,
+    isSelected: Boolean,
+    onClick: () -> Unit
+) {
+
+    val contentColor = when (isSystemInDarkTheme()) {
+        true -> PhotonColors.LightGrey10
+        false -> if (isSelected) PhotonColors.LightGrey10 else PhotonColors.DarkGrey90
+    }
+
+    val backgroundColor = when (isSystemInDarkTheme()) {
+        true -> if (isSelected) PhotonColors.Violet50 else PhotonColors.DarkGrey50
+        false -> if (isSelected) {
+            PhotonColors.Ink20
+        } else {
+            // Custom color code matching the Figma design.
+            @SuppressWarnings("MagicNumber")
+            val lightBackground = Color(0x14312A65)
+            lightBackground
```

Comment:
Can we just return Color(0x14312A65) here? Do we also have a name for this color in the Figma design?

Reply:
That assignement allows for the `MagicNumber` suppression there instead of suppressing a bit higher up.

Seems like the recent `PhothonColors` update changed the definition of `Ink20` and that color code needs to be hardcoded also (selected background used in light theme).

`14312A65` is `Ink80` but with just 8% alpha so instead of hardcoding that value I can use `PhotonColors.Ink80.copy(alpha = 0.08f)` which is still not great since we're still adding a custom color but makes the relation with the `PhotonColors` clearer.

Reply:
I think it is also fine to add `Ink80A8` in `PhotonColors` as well. We already do this in our existing colors.xml

Reply:
I would stay away as much as possible of adding new colors, dimensions, etc on top on what we have to avoid resources blowing up and be hard to keep track of/maintain like it's the situation now.
Google recommends a very small subset of colors etc with a reason and if we're also starting fresh with the goal of a cohesive design and reusability we should strive to stick to our original specs which needs to be stable.
So I'd prefer having something ugly in code that immediately screams custom to easily know that ain't right and needs to be updated.

I see this custom color as rather a sign that it needs a different value from the ones we have in [PhotonColors](https://github.com/mozilla-mobile/android-components/blob/50914d96379f6ef6cfbc13e2965c6e884247e90a/components/ui/colors/src/main/java/mozilla/components/ui/colors/PhotonColors.kt). @topotropic

Reply:
Went with the ugly custom color codes.